pallets/xcm-transactor/src/lib.rs

```diff
@@ -188,15 +187,15 @@ pub mod pallet {
 		/// Extra weight involved when transacting without DescendOrigin
 		/// This should always be possible in a destination chain, since
 		/// it involves going through the sovereign account
-		pub transact_extra_weight: XcmV2Weight,
+		pub transact_extra_weight: Weight,
 		/// Max destination weight
-		pub max_weight: XcmV2Weight,
+		pub max_weight: Weight,
 		/// Whether we allow transacting through signed origins in another chain, and
 		/// how much extra cost implies
 		/// Extra weight involved when transacting with DescendOrigin
 		/// The reason for it being an option is because the destination chain
 		/// might not support constructing origins based on generic MultiLocations
-		pub transact_extra_weight_signed: Option<XcmV2Weight>,
+		pub transact_extra_weight_signed: Option<Weight>,
```

Comment:
I think this requires migration :(. There is an alternate way in which we can do this though, to bypass migration:

- we dont modify the struct that goes into storage, but we append the default_proof_size when we send it to the relay